### PR TITLE
limit House to places where it is present

### PR DESCRIPTION
**Pull request by matkoniecz**
based on https://pl.wikipedia.org/wiki/House_(marka)
reported in https://github.com/streetcomplete/StreetComplete/issues/3774#issuecomment-1045137914

Note: edited online, not tested

### CODE REVIEW

**Comment by matkoniecz** [comment]
Fun fact: someone run undiscussed bot edit removing `name=House` as unwanted descriptive names. Including shop names of that brand.

**Comment by arch0345** [comment]
Thanks @matkoniecz

File: data/brands/shop/clothes.json [hunk 1]
```diff
@@ -3306,7 +3306,7 @@
     {
       "displayName": "House",
       "id": "house-3937bd",
-      "locationSet": {"include": ["001"]},
+      "locationSet": {"include": ["pl", "ru", "ua", "cz", "ro", "sk", "ee", "bg", "hr", "lt", "lv", "hu", "rs", "kz", "fi", "si", "ba"]},
```
Comment (peternewman) [inline data/brands/shop/clothes.json:3309]: A bit late, but do we want the locationSet list sorted to make it easier for others in future? I'm not sure if the CI currently checks for duplicates...

Or maybe the build command sorts it for us (or could do)?

Comment (bhousel) [reply data/brands/shop/clothes.json:3309]: Yes I think the build command will sort it.

Comment (peternewman) [reply data/brands/shop/clothes.json:3309]: Indeed it did:
https://github.com/osmlab/name-suggestion-index/commit/cb2dabf364d2d5ac1f3e7580cab68ec45b3bc1b2#diff-682a76e42ab2b82dbdfe83f676ce59a3cfa6a81751350a288d8c30eb1f49a4b0R3308

**Pull request merged**